### PR TITLE
[Fix "Card Info"] Updated strings from Crowdin

### LINKED ISSUE #7793: [Bug] In languages other than English, the latter half of the gesture-triggered action menu causes malfunction
###### Reproduction Steps

1. In `Settings - AnkiDroid - Language`, select a language other than English, e.g. German, French, Japanese, Chinese simplified.

  Then, in the language, 

2. Select `Settings - Gestures - Enable gestures`.
3. Tap `Swipe up`.
4. Tap `Bury note` in the menu.

   * Strictly speaking, to make sure, tap the item whose string should correspond to `Bury note`.

       e.g. 
                Tap `Notiz aufschieben`
                Tap `Enterre la note`
                Tap `ノートを延期`
                Tap `隐藏笔记`

5. Swipe up on the study screen.

###### Expected Result

- The next card is displayed.

- Then, if you select `English` in `Language`, and re-open the `Swipe up` action menu, you see `Bury note` selected, of course.

###### Actual Result

- The learning is aborted, and the home screen (the Decks screen) is displayed.

- Then, if you select `English` in `Language`, and re-open the `Swipe up` action menu, you see `Abort learning` selected.

image

###### Debug info

- Ver.2.14.0

- The upper 10 items in the menu (the items above the blue line in the following image) work properly.

image

- In languages other than English and Japanese, the string which should correspond to `Card info` doesn't exist. 

    (In Japanese, it didn't either, but I added it on November 8. The Japanese string which should correspond to `Lookup expression` has gone missing instead.) 

- Instead of the string which should correspond to `Card info`, the string which should do to `Tag note` actually correspond to `Card info`. In other words, the translated string for `Tag note` has slid up to the position of `Card info`.

- This "sliding up by one step" continues until the item whose string should correspond to `Page Down`, i.e. until the third item from the bottom (the item on the green line in the image). 

- The second item from the bottom, whose string also correspond to  `Page Down`, works properly, or the item hasn't been translated (e.g. in German).

- This discordance has originally occurred on Crowdin for some reason.  

###### Research

- [x] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid
- [x] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [x] I have searched for similar existing issues here and on the user forum
- [x] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

### PR DESCRIPTION
**Pull request by david-allison-1**
Fixes #7793 - Card Info was in the wrong place in many languages.

This is a draft - we need another sync in half an hour

### CODE REVIEW

File: AnkiDroid/src/main/res/values-eo/11-arrays.xml [hunk 1]
```diff
@@ -50,7 +50,8 @@
         <item>Respondo pli bona ol konsilinda</item>
         <item>Malfari</item>
         <item>Redakti noton</item>
-        <item>Etikedi noton</item>
+        <item>Card Info</item>
```
Comment (david-allison-1) [inline AnkiDroid/src/main/res/values-eo/11-arrays.xml:53]: I've fixed this issue in CrowdIn

File: AnkiDroid/src/main/res/values-ja/11-arrays.xml [hunk 2]
```diff
@@ -51,6 +51,7 @@
         <item>元に戻す</item>
         <item>ノートを編集</item>
         <item>カード情報</item>
```
Comment (david-allison-1) [inline AnkiDroid/src/main/res/values-ja/11-arrays.xml:53]: This is also incorrect

Comment (david-allison-1) [reply AnkiDroid/src/main/res/values-ja/11-arrays.xml:53]: @snowtimeglass There seems to be a lot of inconsistencies with the Japanese translations. My fixes that worked for other languages appear problematic in Japanese (via Google Translate).

Something like - "カードの表示を中止" - "delete note" doesn't seem correct.

Could you let us know when you've gone over these values and ensured they're consistent.

Comment (snowtimeglass) [reply AnkiDroid/src/main/res/values-ja/11-arrays.xml:53]: Thank you. I've checked the values, and added 2 translations in the following pages.

Delete note
https://crowdin.com/translate/ankidroid/all/en-ja?filter=basic&value=3#6534946

Bury note
https://crowdin.com/translate/ankidroid/all/en-ja?filter=basic&value=3#6534952

Thanks to your help, they're consistent now. I've ensured it.

**Comment by david-allison-1** [comment]
Rerun: https://github.com/ankidroid/Anki-Android/actions/runs/390884234

**Comment by david-allison-1** [comment]
Unsure about whether we can do this for 2.14.2 since it's a translation fix and translations have diverged.

**Comment by mikehardy** [comment]
Translations have not diverged in backwards incompatible ways, I'm still picking i18n syncs to 2.14.x

**Review by mikehardy — APPROVED** [review]

**Pull request merged**